packages/app/src/systems/Bridge/components/NetworkConnectCard/NetworkConnectCard.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Flex } from '@fuel-ui/react';
+
+import { NetworkConnectCard } from './NetworkConnectCard';
+
+export default {
+  component: NetworkConnectCard,
+  title: 'NetworkConnectCard',
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export const Usage = () => {
+  return (
+    <Flex align="center" justify="center">
+      <NetworkConnectCard
+        networkName="Fuel"
+        networkImageUrl="public/fuel-logo.svg"
```

Comment:
The logo of Fuel does not appear in the storybook. You can import the image using `import imageSrc from '../../public/fuel-logo.svg'`.